### ISSUE TITLE
Enhance Website UX with Animation

### ISSUE THREAD

**Issue by ashutoshgithubs** [ashutoshgithubs/iemhacks_DEV-G#3]
This website could benefit from incorporating animation to improve user engagement and interaction. Animated elements can capture attention, provide feedback, and create a more dynamic user experience.

**Comment by ayan-joshi** [comment]
I would like to work on this issue under gssoc24

**Comment by Chetan8299** [comment]
I can add minimalistic animations to the website using GSAP and smooth scrolling with locomotive js to improve UX of the website so please assign this issue to me to contribute under GSSOC'24

**Comment by Kxrishx03** [comment]
Hii I can add some smooth animations in website. Please assign me this issue. @ashutoshgithubs

**Comment by ashutoshgithubs** [comment]
Hi @ayan-joshi , Are you ready to work on this issue/enhancement?? Let me know here.

**Comment by ayan-joshi** [comment]
> Hi @ayan-joshi , Are you ready to work on this issue/enhancement?? Let me know here.

Yes @ashutoshgithubs , just can u explain further more what kind of animation or I can add it by my knowledge

**Comment by ashutoshgithubs** [comment]
@ayan-joshi Do  you know framer motion?

**Comment by KavyaBS123** [comment]
Hey @ashutoshgithubs can you please assign his issue to me I would love to work on it.

**Comment by ayan-joshi** [comment]
> @ayan-joshi Do you know framer motion?

@ashutoshgithubs yes I can work with it

**Comment by ashutoshgithubs** [comment]
Hi @ayan-joshi , You are assigned this issue to solve. Best wishes!

**Comment by ashutoshgithubs** [comment]
Hi @ayan-joshi any work progress on this issue??

**Comment by ayan-joshi** [comment]
> Hi @ayan-joshi any work progress on this issue??

@ashutoshgithubs Hello , I'm working on this taking long as my exams are going on , I'll create the pr in some days
